src/components/NewMessage/NewMessage.vue

```diff
@@ -507,10 +507,12 @@ export default {
 			}
 
 			// FIXME upstream: https://github.com/nextcloud-libraries/nextcloud-vue/issues/4492
-			const temp = document.createElement('textarea')
-			temp.innerHTML = this.text.replace(/&/gmi, '&amp;')
-			this.text = temp.value.replace(/&amp;/gmi, '&').replace(/&lt;/gmi, '<')
-				.replace(/&gt;/gmi, '>').replace(/&sect;/gmi, '§')
+			if (this.text !== '') {
```

Comment:
```suggestion
			if (this.hasText) {
```
it's trimmed in addition